pom.xml

```diff
@@ -70,7 +70,7 @@
         <repository>
             <id>spring-milestone</id>
             <name>Spring Milestones</name>
-            <url>http://repo.spring.io/libs-milestone</url>
+            <url>https://repo.spring.io/libs-milestone</url>
```

Comment:
@rwinch I though you dropped support for HTTP on January 15th, why did this still work?